pkg/service/replay/replay.go

```diff
@@ -28,10 +28,10 @@ var totalTestFailed int
 
 // emulator contains the struct instance that implements RequestEmulator interface. This is done for
 // attaching the objects dynamically as plugins.
-var emulator RequestEmulator
+var requestMockemulator RequestMockHandler
 
-func SetTestUtilInstance(instance RequestEmulator) {
-	emulator = instance
+func SetTestUtilInstance(emulatorInstance RequestMockHandler) {
```

Comment:
I think you are not using this function. If this doesn't have any need please remove it

Reply:
this is being used

Reply:
Can point out the code where it is being used in this repo..?